tests/integration/utility_test_functions.py

```diff
@@ -9,10 +9,11 @@
 from cromshell.utilities import cromshellconfig
 
 
-def run_cromshell_command(command: list, exit_code: int):
+def run_cromshell_command(command: list, exit_code: int, options: list = None):
```

Comment:
Let's document what these parameters are.

Reply:
done, also changed name to subcommand_options